docs/source/API/core/builtinreducers/BOr.rst

```diff
@@ -47,50 +47,60 @@ Synopsis
             BOr(const result_view_type& value_);
     };
 
-Public Class Members
---------------------
+Interface
+---------
+
+.. cppkokkos:class:: template<class Scalar, class Space> BOr
+
+    .. rubric:: Public Types
+
+    .. cppkokkos:type:: reducer
+
+        The self type
+
+    .. cppkokkos:type:: value_type
 
-Typedefs
-~~~~~~~~
+        The reduction scalar type.
 
-* ``reducer``: The self type.
-* ``value_type``: The reduction scalar type.
-* ``result_view_type``: A ``Kokkos::View`` referencing the reduction result 
+    .. cppkokkos:type:: result_view_type
 
-Constructors
-~~~~~~~~~~~~
+        A ``Kokkos::View`` referencing the reduction result
 
-.. cppkokkos:kokkosinlinefunction:: BOr(value_type& value_);
+    .. rubric:: Constructors
 
-    * Constructs a reducer which references a local variable as its result location.
+    .. cppkokkos:kokkosinlinefunction:: BOr(value_type& value_);
 
-.. cppkokkos:kokkosinlinefunction:: BOr(const result_view_type& value_);
+        Constructs a reducer which references a local variable as its result location.
 
-    * Constructs a reducer which references a specific view as its result location.
+    .. cppkokkos:kokkosinlinefunction:: BOr(const result_view_type& value_);
 
-Functions
-~~~~~~~~~
+        Constructs a reducer which references a specific view as its result location.
 
-.. cppkokkos:kokkosinlinefunction:: void join(value_type& dest, const value_type& src) const;
+    .. rubric:: Public Member Functions
 
-    * Store logical ``or`` of ``src`` and ``dest`` into ``dest``:  ``dest = src | dest;``.
+    .. cppkokkos:kokkosinlinefunction:: void join(value_type& dest, const value_type& src) const;
 
-.. cppkokkos:kokkosinlinefunction:: void init(value_type& val) const;
+        Store logical ``or`` of ``src`` and ``dest`` into ``dest``:  ``dest = src | dest;``.
 
-    * Initialize ``val`` using the ``Kokkos::reduction_identity<Scalar>::land()`` method. The default implementation sets ``val=0``.
+    .. cppkokkos:kokkosinlinefunction:: void init(value_type& val) const;
 
-.. cppkokkos:kokkosinlinefunction:: value_type& reference() const;
+        Initialize ``val`` using the ``Kokkos::reduction_identity<Scalar>::land()`` method. The default implementation sets ``val=0``.
 
-    * Returns a reference to the result provided in class constructor.
-    
-.. cppkokkos:kokkosinlinefunction:: result_view_type view() const;
+    .. cppkokkos:kokkosinlinefunction:: value_type& reference() const;
+
+        Returns a reference to the result provided in class constructor.
+
+    .. cppkokkos:kokkosinlinefunction:: result_view_type view() const;
+
+        Returns a view of the result place provided in class constructor.
 
-    * Returns a view of the result place provided in class constructor.
-   
 Additional Information
 ~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
we should use ^^^^^^ here (see this https://github.com/kokkos/kokkos-core-wiki/pull/385/files)